pkg/apiserver/apiserver.go

```diff
@@ -65,8 +65,7 @@ import (
 	clusterkapisv1alpha1 "kubesphere.io/kubesphere/pkg/kapis/cluster/v1alpha1"
 	configv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/config/v1alpha2"
 	crd "kubesphere.io/kubesphere/pkg/kapis/crd"
-	devopsv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/devops/v1alpha2"
-	devopsv1alpha3 "kubesphere.io/kubesphere/pkg/kapis/devops/v1alpha3"
+	kapisdevops "kubesphere.io/kubesphere/pkg/kapis/devops"
```

Comment:
Personally, I prefer to organize APIs in the different groups like the reset, it's easy to tell which versions we are using.

Reply:
+1 put all the DevOps-related things into one place.